src/main/kotlin/org/rust/ide/colors/RustColors.kt

```diff
@@ -19,6 +19,11 @@ enum class RustColor(val humanName: String, externalName: String, fallback: Text
     STRING                ("String", "org.rust.STRING", Default.STRING),
     NUMBER                ("Number", "org.rust.NUMBER", Default.NUMBER),
 
+    STRUCT                ("Struct", "org.rust.STRUCT", Default.CLASS_NAME),
+    TRAIT                 ("Trait",  "org.rust.TRAIT",  Default.INTERFACE_NAME),
+    ENUM                  ("Enum",   "org.rust.ENUM",   Default.CLASS_NAME),
+    MODULE                ("Module", "org.rust.MODULE",   Default.IDENTIFIER),
```

Comment:
Sorry for nitpicking, but spacing is inconsistent here :smile: Also I'd name these colours `STRUCT_NAME` etc. instead as such naming is more meaningful imho.